packages/core/admin/admin/src/content-manager/components/BlocksInput/BlocksInput.tsx

```diff
@@ -0,0 +1,65 @@
+import * as React from 'react';
+
+import { Box, Flex, Typography } from '@strapi/design-system';
+import { type MessageDescriptor, useIntl } from 'react-intl';
+import styled from 'styled-components';
+
+// @ts-expect-error TODO convert to typescript
+import { Hint } from '../Hint';
+
+import { BlocksEditor } from './BlocksEditor';
+
+const TypographyAsterisk = styled(Typography)`
+  line-height: 0;
+`;
+
+const LabelAction = styled(Box)`
+  svg path {
+    fill: ${({ theme }) => theme.colors.neutral500};
+  }
+`;
+
+interface BlocksInputProps extends Omit<React.ComponentProps<typeof BlocksEditor>, 'ref'> {
```

Comment:
Isn't there a `ComponentPropsWithoutRef` to save you doing this? 🤔

Reply:
nice, i didn't know about it